spec/schema.json

```diff
@@ -320,6 +448,70 @@
         }
       }
     },
+    "GeneralPartnerData": {
+       "type": "object",
+       "properties": {
+        "country": {
+          "type": "string"
+        },
+        "date_effective_from": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "date_of_birth": {
+         "$ref": "#/definitions/DateOfBirth"
+        },
+        "etag": {
+          "type": "string"
+        },
+        "forename": {
+          "type": "string"
+        },
+        "former_names": {
+          "type": "array",
+          "items": {
+            "type": "string"
```

Comment:
Is former names going to be an array? Is this equivalent to other names in roe? If so that was just a single string.
Also is it "former" or "previous" names?